src/untangled_spec/reporters/impl/diff.cljc

```diff
@@ -42,7 +42,7 @@
           (let [d (diff ev av :recur)
                 diffs (cond
                         (diff-elem? d) (assoc diffs [k] d)
-                        (diff? d) (map-keys #(vec (cons k %)) d)
```

Comment:
Looks fine, but i would say that we might have a similar problem here: https://github.com/jasonjckn/untangled-spec/blob/a52fdea822a7939cee4619192d8a123a3e6129e7/src/untangled_spec/reporters/impl/diff.cljc#L61